include/unicorn/unicorn.h

```diff
@@ -273,6 +273,15 @@ typedef enum uc_query_type {
     UC_QUERY_PAGE_SIZE,
 } uc_query_type;
 
+// Metadata stub for the variable-size cpu context used with uc_context_*()
+typedef struct uc_context {
+   uc_arch arch;
+   uc_mode mode;
+   size_t size;
+   bool used;
```

Comment:
i have some concerns about this struct, but anyway i think user should be responsible for how uc_context_t is used, but should not push this to the framework with this "used" field.